R/derive_extreme_event.R

```diff
@@ -97,6 +97,8 @@
 #'       1. Only the variables specified for the `keep_source_vars` field of the
 #'       event, and the by variables (`by_vars`) and the variables created by
 #'       `set_values_to` are kept.
+#'       1. If `keep_source_vars = NULL` is used for an event in `derive_extreme_event()`
+#'       the value of the `keep_source_vars` argument of `derive_extreme_event()` is used.
```

Comment:
I would not add it as separate item but add it to the previous one:
```suggestion
#'       If `keep_source_vars = NULL` is used for an event in `derive_extreme_event()`,
#'       the value of the `keep_source_vars` argument of `derive_extreme_event()` is used.
```

Reply:
Changes made and pushed. Its going through the checks now.